lib/cmds/init/success.ts

```diff
@@ -61,8 +62,5 @@ export default async function success({
     )
   }
 
-  console.log(
-    'Do you want to give us feedback on the experience of using this command?'
-  )
-  console.log(`Share your thoughts at ${FEEDBACK_LINK}`)
+  console.log(`Share your feedback at ${FEEDBACK_LINK}`)
```

Comment:
I think it could be clearer this is feedback on the CLI init command only.

Reply:
This should be for the CLI as a whole, we need less noise with the code examples :)

Reply:
I agree with that.

Reply:
```suggestion
  console.log(`Share your feedback for this CLI at ${FEEDBACK_LINK}`)
```
Maybe?

Reply:
Isn't the context quite obvious though?

Reply:
Maybe I'm dumb but I don't think it is to me 😆

Reply:
😊 the whole flow will be reviewed by content writers anyway. Feel free to suggest a commit